client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/useInteractionsTab.ts

```diff
@@ -26,7 +32,34 @@ const useInteractionsTab = (props: useInteractionsTabInput) :  useInteractionsTa
     }
 
     const loadInteractions = () => {
-        // TODO: add loading from DB
+        axios.post('/intersections/getContactEvent', {
+            investigationId: epidemiologyNumber
+        }).then((result) => {
+            const fetchedContactEvents: InteractionEventDialogData[] = result.data.map((eventFromServer: any) => {
+                for(let key of Object.keys(eventFromServer)) {
+                    if(eventFromServer[key] === null) {
+                        eventFromServer[key] = undefined;
+                    }
+                    if(key === 'contacts') {
+                        const contactsFromServer = eventFromServer[key];
+                        let contactsOfEvent: Contact[] = [];
+                        if(contactsFromServer.length > 0) {
+                            contactsFromServer.forEach((singleContact: any) => {
```

Comment:
singleContact: Contact

Reply:
Or instead allowing optional properties in the Contact interface